lib/src/components/Sections/index.vue

```diff
@@ -880,7 +917,7 @@ export default {
       headers: sectionHeader(((inBrowser) ? {} : {origin: this.$sections.projectUrl})),
     };
 
-    const URL = `${this.$sections.serverUrl}/project/${this.$sections.projectId}/page/${this.pageName}`;
+    const URL = `${this.$sections.serverUrl}/project/${this.$sections.projectId}/page/${encodeURIComponent(this.pageName)}`;
```

Comment:
you should rely on the pagePath as the pageName can be different than the path

Reply:
Actually here I am using the pageName prop of the sections main component for both `pageName` and `pagePath`.

When a user declare a static Section page on his Website, He will be setting the pageName of his section through the pageName prop

<img width="399" alt="Screenshot 2023-06-30 at 9 30 22 AM" src="https://github.com/Geeks-Solutions/nuxt-sections/assets/101403093/40a6d0d6-a3cd-420c-bf23-87921b15cc10">

When Sections is using the dynamic dynamic page feature once path is updated from the settings.
In the dynamic page `_.vue` code I am feeding the pageName prop with the value of the path `pageName: this.$route.params.pathMatch ? this.$route.params.pathMatch : '/',`

I think this make much more sense while keeping support for the same prop used and without declaring a new pagePath prop that might force us to set conditions on the URLs and change the pageName prop everywhere it is being used